apps/typegpu-docs/src/content/examples/rendering/perlin-noise/index.ts

```diff
@@ -25,6 +25,24 @@ const gridSizeAccess = tgpu['~unstable'].accessor(d.f32);
 const timeAccess = tgpu['~unstable'].accessor(d.f32);
 const sharpnessAccess = tgpu['~unstable'].accessor(d.f32);
 
+const exponentialSharpen = tgpu['~unstable'].fn([d.f32, d.f32], d.f32)(
+  (n, sharpness) => {
+    const sharp = sign(n) * pow(abs(n), 1 - sharpness);
+    return sharp * 0.5 + 0.5;
+  },
+);
+
+const tanhSharpen = tgpu['~unstable'].fn([d.f32, d.f32], d.f32)(
+  (n, sharpness) => {
+    const sharp = tanh(n * (1 + sharpness * 10));
+    return sharp * 0.5 + 0.5;
+  },
+);
```

Comment:
Let's do the normalizing outside of the sharpen functions
```suggestion
const exponentialSharpen = tgpu['~unstable'].fn([d.f32, d.f32], d.f32)(
  (n, sharpness) => sign(n) * pow(abs(n), 1 - sharpness),
);

const tanhSharpen = tgpu['~unstable'].fn([d.f32, d.f32], d.f32)(
  (n, sharpness) => tanh(n * (1 + sharpness * 10)),
);
```